.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,19 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
 1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
 1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
 1. Make sure the branch is deployed: `systemctl --user status scout.target`
-1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
+1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
```

Comment:
Replace the paxa command runned oin hasta with the button at https://pax.scilifelab.se, also above in this list, when the paxa command is called before testing

Reply:
This is intentional: much easier to use the cli when one is anyway on hasta? No need to go to a second system. But you are very right about the above section for cg-vm1!